src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "Users")
```

Comment:
```suggestion
@Table(name = "users")
```
use lower snake case for table names